engine-precompiles/src/native.rs

```diff
@@ -327,18 +354,21 @@ impl<I: IO> Precompile for ExitToNear<I> {
         let (refund_address, mut input) = parse_input(input)?;
         #[cfg(not(feature = "error_refund"))]
         let mut input = parse_input(input)?;
-        let current_account_id = self.current_account_id.clone();
+        #[cfg(not(feature = "ext-connector"))]
```

Comment:
Just wondering, why this check/feature wasn't moved to the `get_eth_connector_contract_account`?

Reply:
Because we don't have access to the `self.current_account_id` inside the `get_eth_connector_contract_account`.